dev-python/snuggs/metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>sci@gentoo.org</email>
+    <name>Gentoo Science Project</name>
+  </maintainer>
+  <maintainer type="project">
+    <email>python@gentoo.org</email>
+    <name>Python</name>
+  </maintainer>
+  <longdescription lang="en">
+  Snuggs are s-expressions for Numpy
+</longdescription>
```

Comment:
Pkgcheck says:
```
  dev-python/snuggs
    RedundantLongDescription: metadata.xml longdescription closely matches DESCRIPTION
```

Reply:
this was intentionally, but now I found a better description from debian.